support-workers/src/test/scala/com/gu/support/workers/JsonFixtures.scala

```diff
@@ -246,6 +253,17 @@ object JsonFixtures {
       }
     """
 
+  val sundayPaperJson =
+    """
+      {
+        "productType": "Paper",
+        "currency": "GBP",
+        "billingPeriod" : "Monthly",
+        "fulfilmentOptions" : "HomeDelivery",
+        "productOptions" : "Sunday"
```

Comment:
```suggestion
        "billingPeriod": "Monthly",
        "fulfilmentOptions": "HomeDelivery",
        "productOptions": "Sunday"
```
slightly odd formatting here